Add classifier_activation param to applications

ivallesp opened the pull request:
I have noticed that the DenseNet and NasNet models did not have the possibility to specify the `classifier_activation` parameter, when the main classes have it implemented and the other models in `tf.keras.applications` also have them available, hence I have added those parameters. I have tested the code locally.

ivallesp committed:
Add classifier_activation to applications + tests

fchollet reviewed (changes requested):
Thank you for the PR! Please add a unit test to check this change. Also, you may need to update the API pbtxt files since this is an API change.

ivallesp commented:
I have added the unit tests and squashed the commit with the previous one. Could you please elaborate on the pbtxt request? What do you need me to change?

fchollet reviewed (approved):
LGTM, thank you

fchollet commented:
> Could you please elaborate on the pbtxt request? What do you need me to change and where?

I've checked that we should be able to handle that on our side.

rthadur commented:
@ivallesp can you please check sanity build failures ?

tensorflowbutler commented:
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

gbaned commented:
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
